### DISCUSSION TITLE
[Question] What is different with VSCode extension

### DISCUSSION THREAD

**Discussion by chc7042** [dutchigor/pluggable-electron#7] [General]
Suddenly, a question arose about the difference from plugin development using vscode. In the case of VSCode, the plugin is created in the location c:\users\user\.vscode\extension, and it is thought that the plugin works by referring to the location. Is there any difference from the extension structure of VSCode?

**Comment by dutchigor** [comment]
This is very similar in PLuggable Electron and actually somewhat based on VSCode. in Pluggable Electron, the plugins folder is configurable, but the way that it initialises the plugin is very mush the same. E.g. by going through the folders in the plugins folder and checking the pakcage.json file for an extensions entry in each plugin.
